js/finder.js

```diff
@@ -178,7 +178,10 @@ const finder = (function () {
   }
 
   function attr(input) {
-    const attrs = Array.from(input.attributes).filter((attr) => config.attr(attr.name, attr.value));
+    const order = { id: 1 };
+    const attrs = Array.from(input.attributes)
+      .filter((attr) => config.attr(attr.name, attr.value))
+      .sort((a, b) => ((order[a.name] || 2) - (order[b.name] || 2)));
```

Comment:
If we're modifying this finder.js library, it might be good to add a comment at the top that we've modified it, so we don't accidentally transpile a new version someday and clobber these changes.